website/docs/docs/deploy/job-scheduler.md

```diff
@@ -0,0 +1,81 @@
+| Prep time | Amount of time it takes for dbt Cloud to get the job ready to run your project on your cloud data platform. |
+| Run | A single unique attempt to run a job. |
+| Run slot | Run slots control the number of jobs that can run concurrently: one run slot allows for a single job at a time. Adding more run slots enables multiple jobs to run at the same time. |
+| Scheduler | The dbt Cloud application that controls job execution. |
+| Threads | When you run dbt, it creates a graph of links between models. The number of [threads](/docs/core/connection-profiles#understanding-threads) is the maximum number of paths that dbt can work on at once. The default value is 4 threads. |
+| Wait time | Amount of time that dbt Cloud waits before running your job, either because there are no available slots for running jobs or because a previous run of the same job is still in progress. |
+
+
+## Scheduler queue
+
+The dbt Cloud scheduler queues a deployment job to be processed when it's triggered to run, either by schedule or by API. 
+
+Before the job starts executing, the scheduler checks two conditions to determine if the run can start executing:
+
+- **The account must have an available run slot** &mdash; If all run slots are occupied, the queued run will wait. The wait time is displayed in dbt Cloud. If there are long waiting times, [upgrading to enterprise](https://www.getdbt.com/contact/) can provide more run slots and allow for higher job concurrency.
+
+- **There must not be a run of the same job already in-flight** &mdash; The scheduler executes distinct runs of the same dbt Cloud job serially to avoid model build collisions. If there's a job already running, the queued job will wait and the waiting time will be displayed in dbt Cloud.
+
+Once the available slot conditions and distinct run checks are met, the scheduler will prepare the job to run on your data platform. The time it takes to prepare the job is displayed as **prep time.**
+
+Collectively **wait time** and **prep time** is the time a run spends in queue.
+
+<Lightbox src="/img/docs/dbt-cloud/deployment/deploy-scheduler.jpg" width="85%" title="An overview of a dbt Cloud job run"/>
+
+## Job memory
+
+In dbt Cloud, each run of a job gets its own [Kubernetes pod](https://en.wikipedia.org/wiki/Kubernetes) with a default megabyte (MB) memory limit. If a job uses up all of that memory, the pod is terminated with a "memory error" message. Each pod has its own memory limit, so having multiple jobs that use up some of the MBs each won't exhaust MB limit set for your account.
+
+It's possible that jobs can consume a lot of memory in various situations, such as when bringing data back into dbt for processing, running a high thread count, or retrieving tables from schemas during docs generation.
+
+
+<!-- leaving space for CI job treatment, runtime, warm up pods updates
+
+-->
+
+## Run cancelation
```

Comment:
will this section have more content related to different types of cancellations?